_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty](#non-empty) `autocomplete` attribute, except if one of the following is true:
+
+- The element is not [visible on the page](#visible-on-the-page) and not [exposed to assistive technologies](#exposed-to-assistive-technologies)
+- The element is an `input` element with a `type` property of `hidden`, `button`, `submit` or `reset`
+- The element has a `disabled` or `aria-disabled="true"` attribute
+- The element has `tabindex="-1"` and has a [semantic role](#semantic-role) that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
+
+### Expectation 1
+
+The `autocomplete` attribute is a single term, or a space seperated list of terms.
+
+### Expectation 2
+
+The autocomplete term(s) follow the [HTML 5.2 specification](https://www.w3.org/TR/html52/sec-forms.html#autofill-detail-tokens), which requires that it/they match the following:
+
+1. Has a section-* value _(optional)_
+2. Has either "shipping" or "billing" _(optional)_
+3. Has "home", "work", "mobile", "fax" or "pager" _(optional, only for "email", "impp", "tel" or "tel-*")_
+4. Has a [correct autocomplete field](#correct-autocomplete-field) _(required)_
+
```

Comment:
Entries 3 and 4 are exclusive and it is required to have one of them. That is not how it reads currently

Reply:
It isn't. 4 is required, 3 is only allowed for certain cases.

Reply:
Are you sure? Because what's in the spec is "Either of the following two options", referring to this list's options 3 and 4. I interpret "either" as an exclusive or, if that is the problem